query_tests/generate_rnd_queries.py

```diff
@@ -327,6 +335,18 @@ def expr_with_operator(column, provider, inner_type, dimensions):
         return f'{column} IS NULL'
     else:
         op = random.choice(operators)
+        if op in ADDITIONAL_STRING_OPERATORS:
```

Comment:
Wouldn't it make more sense to use the `scalar` abstraction instead of hard-coding operator specific logic here?